### PR TITLE
React Capstone Project: Rwanda Locations App

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, please review this PR.

Changes made:

- Fetch Rwanda locations data from API
- Display the Provinces and metrics on cards
- Fetch selected Province data from API
- Display the Districts in a List
- Show various count data in header
- Use Redux to fetch and store data
- Add tests for app
- Implement design given using Bootstrap
- Deploy website to this [Link](https://rwanda-locations.onrender.com/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { useEffect } from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
-import Provinces from './components/Provinces';
-import Districts from './components/Districts';
+import Provinces from './components/Province/Provinces';
+import Districts from './components/District/Districts';
```
Comment (iambenkis) [inline src/App.js:5]: - [x] Keep doing the great job 🚀

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @egichuhis 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ All tests are passing 
✔️ Use Redux 
✔️ The project is a single page application (SPA)

## Suggested changes

- [x] Good job finalizing your React base application 👏  , Good documentation also however a professional README file should not have a broken link. Kindly remove the broken link attached to the logo image and provide a relevant image 👍  or you may also remove the image 👍 

Capture1


- [x] Hey, great job with the project, I admire your effort so far, You've done well. I just wanted to point out something concerning the filtering feature. As it's a requirement to add a filter functionality, so a user could filter by a book name (or other refrence 😄 ), I don't seem to see it working as intended. When a user searches for a province , it should show up what matches the search . Please, try to add this implementation to meet this part of the project requirement. 👍

Capture

Capture

- [ ] Kindly make sure the app has a navbar as the provided has a navbar that is going to help the user to navigate throughout the page 👍 (Eg: Logo,  back button , search button, hamburger button , etc 😄  )  

Capture

Capture

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by egichuhis** [comment]
Thanks @iambenkis for noting that. Cheers!

**Review by V-Blaze — APPROVED** [review]
Hi @egichuhis 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Project deployed and working💯
-  Search featured works fine👍
-  Clean and well-structured code🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**